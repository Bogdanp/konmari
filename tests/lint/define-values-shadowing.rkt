#lang racket/base

(define-values (a b)
  (let ()
    (define a 42)
    (define b 20)
    (+ a b)))

(define (a x)
  x)

b
