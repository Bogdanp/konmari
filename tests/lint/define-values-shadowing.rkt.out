/define-values-shadowing.rkt:9:10:error:identifier 'a' is already defined
/define-values-shadowing.rkt:3:17:warning:identifier 'a' is never used
